tools/coverage_system_tests.rc

```diff
@@ -1,12 +1,26 @@
+; This file controls what modules are examined for code coverage in our system tests.
+
 [run]
 omit =
+    ; Common code, so covered by nifake unit tests.
+    ; Some modules don't even exercise all of _attributes.py and _converters.py, making parts unreachable in system tests.
     */__init__.py
     */_attributes.py
     */_converters.py
+    */errors.py
     */_library_singleton.py
     */_visatype.py
+
+    ; Although the enum names and values vary between modules, all enums are mostly the same. nifake unit tests cover the various kinds that we use.
     */enums.py
-    */errors.py
+
+    ; I really think we shouldn't omit this. It's the public API that all of our system tests call.
```

Comment:
This is an open question. Should we really omit session.py from system test coverage?
I know there are nifake session.py unit tests, but unless all of our fancy functions are covered by unit tests, as well, that doesn't seem like sufficient reason to omit these files.

Reply:
It's the public API that all clients of this package call.

This module's coverage data is probably what you would use to answer the question "are we testing feature X?" I think that usage of coverage data is more important than displaying a percentage in a badge on GitHub. :)

Reply:
Yeah, technically, a lot of that can be determined by looking at interpreter, library coverage, but I agree that we're leaving out important coverage information.

I'm not worried about the inclusion lowering our reported coverage.

Reply:
FWIW: When we first developed nimi-python, we targeted complete code coverage of Session class in our system tests.